### ISSUE TITLE
Ability to obtain products using a currency that does not use sub units

### ISSUE THREAD

**Issue by Gavera** [vendurehq/vendure#1146]
when queyring a product thats is associated to a channel that uses a currency that doenst uses sub-units in practice, for example Chilean Peso (CLP) . The results of the graphql query shows the prices assuming that the specific currency uses sub-units:

![image](https://user-images.githubusercontent.com/18266408/136866467-88d5c66a-3846-457b-bca4-f7aaf187552e.png)

in this screenshot,  the price value of  the first product variant is 2399000 and 2878800 with taxes. On the other hand , in the admin UI  the price value of the product  is 23990,00 (not formatted) and  28,788 with taxes (formatted and the expected value assuming the currency and taxes application).

![image](https://user-images.githubusercontent.com/18266408/136865673-acfdb373-53bf-42b3-89ab-40292e855ed8.png)

Basically, in this case , the results has two extras ceros, leading to inconsistent pricing between the admin UI and the storefront.

PD: Sorry for bad english.
